pandas/core/arrays/base.py

```diff
@@ -123,6 +123,27 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
         """
         raise AbstractMethodError(cls)
 
+    @classmethod
+    def _from_sequence_of_strings(cls, strings, dtype=None, copy=False):
+        """Construct a new ExtensionArray from a sequence of scalars.
+
+        Parameters
+        ----------
+        strings : Sequence
+            Each element will be an instance of the scalar type for this
```

Comment:
IIRC this came up in the discussion of Interval parsing from strings. did we already create some machinery fro this? @jschendel @TomAugspurger @jorisvandenbossche

Reply:
I am not fully sure what you are hinting on. But, the docstring is certainly wrong here (I suppose copy pasted from `_from_sequence`), as here the input should be a sequence of strings, not of scalars of the dtype.

Reply:
Fixed the docstring.

I wasn't able to find anything that added the machinery, but if I'm wrong let me know and I will use it instead of this new method.